src/Core/Abstractions/IStateService.cs

```diff
@@ -168,5 +168,7 @@ public interface IStateService
         Task SetPreLoginEmailAsync(string value);
         string GetLocale();
         void SetLocale(string locale);
+        ConfigResponse GetConfigs();
+        void SetConfigs(ConfigResponse value);
```

Comment:
I think there is a naming mismatch, here `GetConfigs()` is used but on the `IApiService` it's named `GetAllConfigs...()`. IMO both should be with or without the `...All...` in between to be consistent.

Reply:
Removed the "All" to be consistent with the endpoint name.